project/settings/test.py

```
@@ -0,0 +1,10 @@
+from project.settings.base import *  # noqa
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": BASE_DIR / "db.sqlite3",
```

Ruff (0.8.2)

`BASE_DIR` may be undefined, or defined from star imports

(F405)